binding-library/java/pom.xml

```diff
@@ -126,6 +126,9 @@
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-javadoc-plugin</artifactId>
                 <version>${maven-javadoc.version}</version>
+                <configuration>
+                    <doclint>none</doclint>
+                </configuration>
```

Comment:
javadoc was throwing error otherwise because it was thinking that the <String> in OutputBinding<String> is an incomplete tag.